addons/project/models/project_task.py

```diff
@@ -1082,6 +1083,22 @@ def write(self, vals):
         if "personal_stage_type_id" in vals and not vals['personal_stage_type_id']:
             del vals['personal_stage_type_id']
 
+        # sends an email to the 'Task Creation' subtype subscribers
+        # When project_id is changed
+        if vals.get('project_id'):
+            project = self.env['project.project'].browse(vals.get('project_id'))
+            notification_subtype_id = self.env.ref('project.mt_project_task_new').id
+            partner_ids = project.message_follower_ids.filtered(lambda follower: notification_subtype_id in follower.subtype_ids.ids).mapped('partner_id.id')
+            if partner_ids:
+                for task in self:
+                    project_link = task.project_id._get_html_link(title=task.project_id.display_name)
+                    task.message_notify(
+                        body=escape(_('Task Transferred From Project %s')) % project_link,
```

Comment:
```suggestion
                        body=_('Task Transferred from Project %s', project_link),
```
I don't think you need to escape here, since the link is already a Markup object.
Also, the "from" should be lowercase

Reply:
Hello @MissingNoShiny 

changes are applied :slightly_smiling_face: